src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerExecContainer.groovy

```diff
@@ -32,6 +32,10 @@ class DockerExecContainer extends DockerExistingContainer {
     @Optional
     Boolean attachStderr = true
 
+    @Input
+    @Optional
+    String user
```

Comment:
Javadoc with information about format would be valuable I guess.

Reply:
Was just about to say the same :)

Reply:
Thanks for the review feedback, would you want  `@since 3.2.8` within the javadoc as well?

Reply:
sure!